golang/internal/runtime/container/container.go

```diff
@@ -0,0 +1,120 @@
+package containerruntime
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/docker/docker/client"
+	"github.com/hashicorp/go-version"
+	"github.com/rs/zerolog/log"
+)
+
+// If a runtime version doesn't satisfy recommendation: show warning
+// If a runtime version doesn't satisfy minimum: failure\
+
+const (
+	Podman                         = "podman"
+	Docker                         = "docker"
+	MinimumDockerServerVersion     = "20.10.0"
```

Comment:
We need to perform maintenance on this. Don't we want to use environment variables instead of hardcoded numbers? It would be easier to care about these versions. What do you think? cc. @c3ppc3pp and @nandor-magyar

Reply:
This is OK as is, the limitations are given and wont change, so these must be constants.